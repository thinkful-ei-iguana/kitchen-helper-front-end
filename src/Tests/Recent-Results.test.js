import React from 'react';
import ReactDOM from 'react-dom';
import RecentResults from '../Components/Recent-Results';
import { BrowserRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<BrowserRouter>
        <RecentResults />
    </BrowserRouter>, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
    const tree = renderer.create(<BrowserRouter>
        <RecentResults />
    </BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});