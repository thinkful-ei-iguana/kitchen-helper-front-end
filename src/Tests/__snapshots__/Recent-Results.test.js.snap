// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="RecentResults"
>
  <section
    className="flex-container"
  />
</div>
`;
